bindings/Makefile

```diff
@@ -0,0 +1,45 @@
+PROJECT_ROOT := ..
```

Comment:
Seems unnecessary

Reply:
yeah, it used be used in more places. I'll remove